helix-core/src/main/java/org/apache/helix/controller/stages/CustomizedViewAggregationStage.java

```diff
@@ -97,7 +103,8 @@ public void execute(final ClusterEvent event) throws Exception {
       for (Resource resource : resourceMap.values()) {
         try {
           computeCustomizedStateView(resource, stateType, customizedStateOutput, curCustomizedViews,
-              updatedCustomizedViews);
+              updatedCustomizedViews, updatedStartTimestamps);
+          Map<String, CustomizedView> curCustomizedViewsCopy = new HashMap<>(curCustomizedViews);
```

Comment:
As you never use this variable in other places, just move it to line 120.

Reply:
See the next reply.